.github/workflows/unit-test-neuralchat.yml

```diff
@@ -10,11 +10,13 @@ on:
       - 'setup.py'
       - 'intel_extension_for_transformers/llm/finetuning/**'
       - 'intel_extension_for_transformers/llm/quantization/**'
+      - 'intel_extension_for_transformers/llm/runtime/graph/**'
```

Comment:
do we need to run neuralchat UT each time graph changes? especially soon graph will be seperated. how long will this UT cost?

Reply:
This is because Neuralchat reply on graph now. This test will take about half an hour

Reply:
Plus half an hour for optimize UT 😂

Reply:
> This is because Neuralchat reply on graph now. This test will take about half an hour

great, LGTM